ansible/playbooks/aix.yml

```diff
@@ -251,20 +251,57 @@
           stat:
             path: /usr/java8_64
           register: java8
+          when: bootjdk == "openj9"
+          ignore_errors: yes
           tags: java8
 
         - name: Create /usr/java8_64 if it doesn't exist
           file:
             path: /usr/java8_64
             state: directory
-          when: java8.stat.isdir is not defined
+          when:
+            - java8.stat.isdir is not defined
+            - bootjdk == "openj9"
+          ignore_errors: yes
 
         - name: Transfer and Extract Java8
           unarchive:
             src: https://api.adoptopenjdk.net/v2/binary/releases/openjdk8?openjdk_impl=openj9&os=aix&arch=ppc64&release=latest&type=jdk
             dest: /usr/java8_64
```

Comment:
Could `openjdk_impl=openj9` be used as a variable to cut down on the amount of change?

Reply:
Yeah that's a way better way to do it.